src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -161,6 +161,46 @@ button.NormalAnnouncementBubble--deleteMenuItem {
   }
 }
 
+.NormalAnnouncementBubble--translatedLanguages--tooltip {
+  .text--semi-bold();
+  padding-left: @size_xs;
+  padding-right: @size_xs;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--icon {
+  .pathColor(@neutral_medium_gray);
+  .margin--right--2xs();
+  width: 1rem;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--text--desktop {
+  color: @neutral_medium_gray;
+  .text--medium;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--text--mobile {
+  color: @neutral_medium_gray;
+  .text--medium;
+  display: none;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--container {
+  margin-top: 1rem;
```

Comment:
micro nit: we have a mix-in for this (I think it's `.margin--top--l()` but worth double checking)

Reply:
but I also know some folks prefer to not use the mix-ins, so up to you!